miden/tests/integration/stdlib/crypto/falcon.rs

```diff
@@ -1988,7 +1988,7 @@ fn test_falcon512_vector_squared_norm() {
             in_vec[(i << 2) ^ 3],
             in_vec[(i << 2) ^ 2],
             in_vec[(i << 2) ^ 1],
-            in_vec[(i << 2)],
+            in_vec[(i << 2) ^ 0],
```

Comment:
the other option is to remove the parenthesis. I went with this because the compiler should easily remove the noop, and this makes the expression consistent with the ones above.

Reply:
code alignment for the win :laughing:.  Makes sense that the compiler should optimise and in any case this is test code.